src/NuGet.Clients/PackageManagement.UI/Feeds/InstalledPackageFeed.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using NuGet.Packaging.Core;
+using NuGet.Protocol.Core.Types;
+
+namespace NuGet.PackageManagement.UI
+{
+    /// <summary>
+    /// Represents a package feed enumerating installed packages.
+    /// </summary>
+    internal class InstalledPackageFeed : PlainPackageFeedBase
+    {
+        private readonly IEnumerable<PackageIdentity> _installedPackages;
+        private readonly IPackageMetadataProvider _metadataProvider;
+        private readonly Logging.ILogger _logger;
+
+        public InstalledPackageFeed(
+            IEnumerable<PackageIdentity> installedPackages,
+            IPackageMetadataProvider metadataProvider,
+            Logging.ILogger logger)
+        {
+            if (installedPackages == null)
+            {
+                throw new ArgumentNullException(nameof(installedPackages));
+            }
+            _installedPackages = installedPackages;
+
+            if (metadataProvider == null)
+            {
+                throw new ArgumentNullException(nameof(metadataProvider));
+            }
+            _metadataProvider = metadataProvider;
+
+            if (logger == null)
+            {
+                throw new ArgumentNullException(nameof(logger));
+            }
+            _logger = logger;
+
+            PageSize = 25;
```

Comment:
For local do still use a higher page size? I believe this wasn't always 25 in the last version.